src/rpc/blockchain.cpp

```diff
@@ -984,6 +985,37 @@ static RPCHelpMan getblock()
                         }},
                     }},
                 }},
+                    RPCResult{"for verbosity = 3",
+                RPCResult::Type::OBJ, "", "",
+                {
+                    {RPCResult::Type::ELISION, "", "Same output as verbosity = 2"},
+                    {RPCResult::Type::ARR, "tx", "",
+                    {
+                        {RPCResult::Type::OBJ, "", "",
+                        {
+                            {RPCResult::Type::ARR, "vin", "",
+                            {
+                                {RPCResult::Type::OBJ, "", "",
+                                {
+                                    {RPCResult::Type::ELISION, "", "The same output as verbosity = 2"},
+                                    {RPCResult::Type::OBJ, "prevout", "",
```

Comment:
Might want to mention briefly this is only available if there is undo information for the input (e.g., not for coinbase inputs). E.g. a user might be surprised to not see it when running it on the genesis block.

Reply:
Happy to change. It's just about finding the proper sentence. What about 

```cpp
{RPCResult::Type::OBJ, "prevout", "(Only if undo information is available)",
```

?

Or is it too brief?

Reply:
Seems fine to me, agree that the set of conditions is hard to describe precisely in the scope available.